packager/react-packager/src/DependencyResolver/fastfs.js

```diff
@@ -293,16 +293,16 @@ class File {
   }
 
   getFiles() {
-    const files = [];
-    Object.keys(this.children).forEach(key => {
+    const files = Object.keys(this.children).map(key => {
       const file = this.children[key];
       if (file.isDir) {
-        files.push(...file.getFiles());
-      } else {
-        files.push(file);
+        return file.getFiles();
```

Comment:
you both know we have an interview question related to this at FB. Look on the internet for the fastest possible implementation and use that ;)